data_dir/c-s_lwt_basic_multidc.yaml

```diff
@@ -0,0 +1,79 @@
+# LWT test: create and update data using LWT.
```

Comment:
Do you need this profile? If not - please, remove

Reply:
Will remove it.